docs/source/interoperability.rst

```diff
@@ -4,6 +4,38 @@ Interoperability
 DLPack (C)
 ^^^^^^^^^^
 
+Approximate-Nearest-Neighbor Indexes provide an interface to build and search an index via a C API. [DLPack](https://github.com/dmlc/dlpack/blob/main/README.md), a tensor interface framework, is used as the standard to interact with our C API.
```

Comment:
Would be good to add a note that we're using v0.8 ManagedTensor, and will update to 1.0 Versioned one as soon as that's released since MT is deprecated

Reply:
Good point, will add